src/js/keypad/renderKeypad.ts

```diff
@@ -0,0 +1,70 @@
+import { getById } from '../utilities';
+
+
+export const symbolSet: { [key: string]: string } = {
+	Q: "#984",
+	A: "#1126",
+	Lambda: "#411",
+	Lightening: "#990",
+	Alien: "#1132",
+	Hallmark: "#983",
+	BackwardsC: "#1023",
+	BackwardsE: "#1260",
+	PigTail: "#1192",
+	WhiteStar: "#9734",
+	QuestionMark: "#191",
+	Copyright: "#169",
+	SaggyBoobs: "#1148",
+	MirrorK: "#1046",
+	Melting3: "#1134",
+	Sigma: "#1004",
+	CarriageReturn: "#182",
+	B: "#384",
+	PokeyOuteyTongue: "#9786",
+	Trident: "#968",
+	C: "#1022",
+	Caterpillar: "#1134",
+	BlackStar: "#9733",
+	DoubleDagger: "#1154",
+	AE: "#230",
+	N: "#1162",
+	Omega: "#937",
+};
+
+
+
+export function renderKeypad(): void {
+
+	var keypad_inputs = getById("KeypadForm");
+
+	let group = document.createElement("div");
+
+	Object.values(symbolSet).forEach((val, idx) => {
+		let input = document.createElement("input");
+		input.id = `keypad_${idx}`;
+		input.className = `btn-check`;
+		input.setAttribute("type", "radio");
```

Comment:
Hmm. I hadn't thought about it. I guess I should change them.
But I did just figure out why they work. It's because they all have unique `name`  attributes. And that is what determines their 'group'.

Reply:
I'm going to leave them for now because they work, but I'll add a bug for future refactoring into 'toggles'. https://getbootstrap.com/docs/5.2/components/button-group/#checkbox-and-radio-button-groups